docs/docs-content/release-notes/release-notes.md

```diff
@@ -83,11 +83,24 @@ tags: ["release-notes"]
   [Build Provider Images](../clusters/edge/edgeforge-workflow/palette-canvos/build-provider-images.md) for guidance on
   using the **certs** folder to pass multiple proxy certificates to the CanvOS build process.
 
-<!-- prettier-ignore -->
-- The Edge <VersionedLink text="BYOOS" url="/integrations/packs/?pack=edge-native-byoi" />
-  pack has new parameters that allow you to configure node draining behavior during cluster upgrades or repaves.
-  Previously, nodes were always drained during upgrades and repaves, even for single-node clusters. Refer to
-  [Skip Node Draining](../clusters/edge/cluster-management/skip-draining.md) for guidance on configuring draining behavior.
+<!-- prettier-ignore-start -->
+
+- The Edge <VersionedLink text="BYOOS" url="/integrations/packs/?pack=edge-native-byoi" /> pack has new parameters that
+  allow you to configure node draining behavior during cluster upgrades or repaves. Previously, nodes were always
+  drained during upgrades and repaves, even for single-node clusters. Refer to
+  [Skip Node Draining](../clusters/edge/cluster-management/skip-draining.md) for guidance on configuring draining
+  behavior.
+
+  <!-- prettier-ignore-end -->
+
```

Comment:
```suggestion
#### Bug Fixes

```

@lennessyy  This feels like it should be under a Bug Fixes section, right?

Reply:
I debated this because there is no bug fix section for any other product. But I think yeah bug fix does make more sense. I was thinking it was technically already not a bug in 4.5.x. But it did start as a bug, so yes i will move it.